lib/Echidna/Config.hs

```diff
@@ -68,10 +69,12 @@ instance FromJSON EConfig where
         getWord s d = C Dull . fromIntegral <$> v .:? s .!= (d :: Integer)
         xc = liftM4 TxConf (getWord "propMaxGas" 8000030) (getWord "testMaxGas" 0xffffffff)
                            (getWord "maxTimeDelay" 604800)     (getWord "maxBlockDelay" 60480)
+        coverageParser = v .:? "coverage" <&> \case Just True -> Just mempty
```

Comment:
I think the wording `coverageParser` itself can be a bit confusing. I would probably rename it to just `coverage` or `cov` for short.

Reply:
Good point, I was also not sure about this name, changed to `cov`. Now I think it could be even put inline but it is a minor difference.